src/components/RegistrationForm/RegistrationForm.js

```diff
@@ -19,15 +26,16 @@ export default class RegistrationForm extends Component {
   }
 
   nextPage() {
-    this.setState({ currentStep: this.state.currentStep + 1 });
+    this.setState(state => ({ currentStep: state.currentStep + 1 }));
```

Comment:
why you change it?

Reply:
because I need past state and I think this is better way to get one.